### ISSUE TITLE
Give the wand item some special metadata

### ISSUE THREAD

**Issue by Secret-chest** [EngineHub/WorldEdit#2328]
### The Problem

tl;dr: **make the wand you get by doing `//wand` special.**

I believe I'm not the only one who enjoys the convenience of having WorldEdit on the same instance as used to play suvival (cheated survival) normally. It's not ideal, not at all.

By default the wand is a wooden axe. That of course interferes much with survival. I changed it to the creative-only item I find most useless: the `command-block-minecart`. Still, the ideal way would be to make wands only obtainable through their respective commands.

### A Solution

Give some metadata to the items obtained through wand commands, to indicate that they are indeed wands. A regular item of the same type as the wand will no longer work. Basically, make it work similarly to brushes.

### Alternatives

Set the wand to a creative-only item. However, this is not ideal as you still lose access to said item.

**Comment by wizjany** [comment]
duplicates #1306 / #1589

**Comment by Secret-chest** [comment]
> duplicates #1306 / #1589

No, it's not.

**Comment by octylFractal** [comment]
The changes to support this would be the same as those issues. It is effectively a duplicate.
